### ISSUE TITLE
When importing SDF deck, "Deck Img" isn't saved after server reboot.

### ISSUE THREAD

**Issue by StyrmirThorarins** [spacemandev-git/fvtt-card-support#45]
Great module, thanks for sharing!

I came across a bug while setting things up. 

When importing a SDF deck, and assigning a "Deck Img" from the "Import SDF" dialog, everything works as you'd expect during the current session of the VTT server. Dragging the deck onto the background it'll show the correct (the selected) deck image. All the cards show up as they should and appear to work as intended.

The bug comes up when you restart the server / campaign.
After a restart the selected card deck image no longer appears when you drag the deck onto the VTT background. 
Instead it is replaced by the default image of a colored line representation of a card deck.

Checking the imported deck images in the campaigns folders "Deck" folder. The deck image was imported along with the cards into the folder. 
It is simply not showing up anymore when you drag the deck onto the VTT background, as it did before restarting the server.

Tried this with a few decks in several campaigns, the same erroneous behavior happened in all cases.

**Comment by sharkbruhaha** [comment]
What method are you using to import the decks? Checking because just tried several times in my game and it is maintaining the top of deck img. Want to make sure it isn't a particular method.

**Comment by sharkbruhaha** [comment]
Yep, mine have dropped off now too. They are maintained through "Return to Setup" and even going between worlds, but as soon as you close out the application and come back it has no card top.

Devs: I believe it is probably something of you're only looking at the yaml? And the Card Top isn't part of that?
